### PR TITLE
Fix: Remove dev portal entries from 3.5 changelog

### PR DESCRIPTION
**Pull request by lena-larionova**
### Description

As Dev Portal is deprecated on Gateway going forward, we should list any fixes for it in the changelog.

Fixes requested by PM.

### Testing instructions

Preview link: 

### Checklist 

- [x] Review label added 
- [x] PR pointed to correct branch (`main` for immediate publishing, or a release branch: e.g. `release/gateway-3.2`, `release/deck-1.17`)

### CODE REVIEW

**Review by teb510 — APPROVED** [review]

**Review by cloudjumpercat — APPROVED** [review]
LGTM! I'll let you merge this when you're ready

**Pull request merged**
